.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,44 @@
+<!--
+Allo' allo'! 
+Thanks for taking the time to submit an issue.
+Please keep in mind, that GitHub issues are meant to be used for reporting bugs and to
+request new features.
+
+Use Stack Overflow for support: http://stackoverflow.com/questions/tagged/yeoman
+Head over to one of our Gitter rooms (https://gitter.im/yeoman/home) and ask for help if you’re unsure if you ran into a bug or if you have any other question.
```

Comment:
Probably we want to change this reference to Discord :)

Reply:
```suggestion
Head over to our Discord (https://discord.gg/tKnYVm4k) and ask for help in its [`#help` channel](https://discord.com/channels/1326978160515879063/1327268999817334884) if you’re unsure if you ran into a bug or if you have any other question.
```